lib/model/channel.dart

```diff
@@ -1,14 +1,20 @@
+import 'dart:async';
 import 'dart:collection';
 
+import 'package:collection/collection.dart';
 import 'package:flutter/foundation.dart';
 
 import '../api/model/events.dart';
 import '../api/model/initial_snapshot.dart';
 import '../api/model/model.dart';
+import '../api/route/channels.dart';
 import 'realm.dart';
 import 'store.dart';
 import 'user.dart';
 
+// similar to _apiSendMessage in lib/model/message.dart
+final _apiGetChannelTopics = getStreamTopics;
```

Comment:
Could you add a prep commit that renames the existing binding `getStreamTopics` to `getChannelTopics`, then consistently use "channel topics" naming instead of "stream topics" naming?